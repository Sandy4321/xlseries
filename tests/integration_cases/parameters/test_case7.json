{"alignment": "horizontal",
 "blank_rows": "True",
 "composed_headers": "False",
 "data_starts": 2,
 "data_ends": 44,
 "frequency": "Y",
 "headers_coord": ["A8", "A10-A12", "A14-A16", "A18-A22", "A24-A26", "A28", "A30", "A32", "A34", "A36-A39", "A41-A44", "A46", "A48", "A50-A52", "A55"],
 "continuity": "False",
 "missings": "False",
 "missing_value": "None",
 "multifrequency": "False",
 "series_names": "None",
 "time_composed": "False",
 "time_alignment": 0,
 "time_multicolumn": "False",
 "time_format": "int",
 "time_header": "False",
 "time_header_coord": "A6",
 "time_header_next_to_data": "False"}
