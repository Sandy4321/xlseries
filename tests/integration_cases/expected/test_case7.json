{"Current Account":{"694224000000":-5557.7545586227,"725846400000":-8208.889940165,"757382400000":-10981.4529700946,"788918400000":-5104.1722542908,"820454400000":-6755.4126941853,"852076800000":-12115.5339598394,"883612800000":-14464.5980422511,"915148800000":-11909.6446107158,"946684800000":-8954.5178926276,"978307200000":-3780.4352411936,"1009843200000":8701.6535533283,"1041379200000":8073.070841526,"1072915200000":3076.4514721936,"1104537600000":5054.9846214999,"1136073600000":7588.1693674954,"1167609600000":7199.5336767321,"1199145600000":6636.5020859372,"1230768000000":10868.8028258947,"1262304000000":-1465.984636302,"1293840000000":-3712.9436991915,"1325376000000":-1176.4968521628,"1356998400000":-4785.7037797195},"Current Account - Goods (7)":{"694224000000":-1395.9,"725846400000":-2363.6,"757382400000":-4138.9,"788918400000":2357.4,"820454400000":1759.5,"852076800000":-2122.7,"883612800000":-3097.2,"915148800000":-794.6,"946684800000":2451.9,"978307200000":7384.9,"1009843200000":17177.5,"1041379200000":16804.6,"1072915200000":13264.6,"1104537600000":13086.66,"1136073600000":13958.3,"1167609600000":13455.8,"1199145600000":15422.6,"1230768000000":18525.901,"1262304000000":14028.4212171435,"1293840000000":12925.32041082,"1325376000000":15157.77995007,"1356998400000":11119.43458398},"Current Account - Goods (7) - Exports FOB":{"694224000000":12398.9,"725846400000":13268.9,"757382400000":16023.3,"788918400000":21161.7,"820454400000":24042.7,"852076800000":26430.8,"883612800000":26433.7,"915148800000":23308.6,"946684800000":26341.0,"978307200000":26542.7,"1009843200000":25650.6,"1041379200000":29938.8,"1072915200000":34575.7,"1104537600000":40386.8,"1136073600000":46546.2,"1167609600000":55980.3,"1199145600000":70018.8,"1230768000000":55672.052,"1262304000000":68187.201,"1293840000000":84051.130914,"1325376000000":80246.122123,"1356998400000":81659.9959069},"Current Account - Goods (7) - Imports FOB":{"694224000000":13794.8,"725846400000":15632.5,"757382400000":20162.2,"788918400000":18804.3,"820454400000":22283.2,"852076800000":28553.5,"883612800000":29530.9,"915148800000":24103.2,"946684800000":23889.1,"978307200000":19157.8,"1009843200000":8473.1,"1041379200000":13134.2,"1072915200000":21311.1,"1104537600000":27300.14,"1136073600000":32587.9,"1167609600000":42524.5,"1199145600000":54596.2,"1230768000000":37146.151,"1262304000000":54158.7797828565,"1293840000000":71125.81050318,"1325376000000":65088.34217293,"1356998400000":70540.56132292},"Current Account - Services":{"694224000000":-2557.4,"725846400000":-3325.64,"757382400000":-3779.0,"788918400000":-3436.1,"820454400000":-3546.9494237252,"852076800000":-4385.1152374307,"883612800000":-4444.3479741887,"915148800000":-4111.1580558351,"946684800000":-4283.7834719566,"978307200000":-3863.0893563358,"1009843200000":-1460.2197826708,"1041379200000":-1193.101643429,"1072915200000":-1330.9968324096,"1104537600000":-991.5222689589,"1136073600000":-500.6145385161,"1167609600000":-512.7107465331,"1199145600000":-1284.1641971147,"1230768000000":-1285.3424172183,"1262304000000":-1161.15099767,"1293840000000":-2246.9176054867,"1325376000000":-3095.0754869758,"1356998400000":-4104.6943155969},"Current Account - Services - Exports":{"694224000000":2984.3,"725846400000":3070.56,"757382400000":3364.1,"788918400000":3825.6,"820454400000":4405.3546424929,"852076800000":4598.7714713552,"883612800000":4854.0695626126,"915148800000":4718.7187246817,"946684800000":4935.520314834,"978307200000":4627.0530768311,"1009843200000":3495.4017523942,"1041379200000":4499.9139350187,"1072915200000":5288.1462188191,"1104537600000":6634.4347992255,"1136073600000":8022.8590151996,"1167609600000":10363.1121036054,"1199145600000":12155.7815482836,"1230768000000":10966.9967204408,"1262304000000":13646.8642521804,"1293840000000":15609.9328381248,"1325376000000":15269.5763340334,"1356998400000":14767.2127212821},"Current Account - Services - Imports":{"694224000000":5541.7,"725846400000":6396.2,"757382400000":7143.1,"788918400000":7261.7,"820454400000":7952.3040662181,"852076800000":8983.8867087859,"883612800000":9298.4175368013,"915148800000":8829.8767805168,"946684800000":9219.3037867906,"978307200000":8490.1424331669,"1009843200000":4955.621535065,"1041379200000":5693.0155784476,"1072915200000":6619.1430512287,"1104537600000":7625.9570681844,"1136073600000":8523.4735537157,"1167609600000":10875.8228501385,"1199145600000":13439.9457453983,"1230768000000":12252.3391376591,"1262304000000":14808.0152498504,"1293840000000":17856.8504436115,"1325376000000":18364.6518210092,"1356998400000":18871.907036879},"Current Account - Income":{"694224000000":-2393.6545586227,"725846400000":-3071.849940165,"757382400000":-3560.7529700946,"788918400000":-4622.2722542908,"820454400000":-5450.1632704601,"852076800000":-6122.5762942509,"883612800000":-7387.1182700593,"915148800000":-7457.1966197203,"946684800000":-7521.6703686215,"978307200000":-7726.7407023091,"1009843200000":-7555.6351412507,"1041379200000":-8042.5399909624,"1072915200000":-9418.1525253899,"1104537600000":-7524.0337325318,"1136073600000":-6328.9482758849,"1167609600000":-6096.986287778,"1199145600000":-7671.5891068867,"1230768000000":-9081.4388746805,"1262304000000":-13927.5655079773,"1293840000000":-13844.0448177608,"1325376000000":-12758.5053607279,"1356998400000":-11011.3164172626},"Current Account - Income - Investment income":{"694224000000":-2404.6545586227,"725846400000":-3082.549940165,"757382400000":-3571.2529700946,"788918400000":-4634.5722542908,"820454400000":-5459.9632704601,"852076800000":-6118.9762942509,"883612800000":-7382.6182700593,"915148800000":-7468.2966197203,"946684800000":-7529.8140272113,"978307200000":-7730.9407023091,"1009843200000":-7530.7183841938,"1041379200000":-8017.0379224052,"1072915200000":-9378.9267068775,"1104537600000":-7472.6733075766,"1136073600000":-6269.3752527105,"1167609600000":-6025.1022146696,"1199145600000":-7613.6743359665,"1230768000000":-9013.6655456095,"1262304000000":-13865.398020505,"1293840000000":-13789.6393616772,"1325376000000":-12693.4298597375,"1356998400000":-10962.3595094162},"Current Account - Income - Investment income - Interests":{"694224000000":-1480.4772992211,"725846400000":-1581.4357088716,"757382400000":-1771.9153154365,"788918400000":-2540.7070753397,"820454400000":-3379.9727166375,"852076800000":-4199.0411561918,"883612800000":-5089.5410509523,"915148800000":-5852.4954175603,"946684800000":-5920.5438882389,"978307200000":-7472.7086028871,"1009843200000":-7761.0577661307,"1041379200000":-7383.9874299499,"1072915200000":-7091.4919856749,"1104537600000":-3578.0725534743,"1136073600000":-1330.0749662212,"1167609600000":-783.9739508432,"1199145600000":-1519.7085307595,"1230768000000":-2386.1948216472,"1262304000000":-3135.1385979976,"1293840000000":-3055.1629300031,"1325376000000":-3500.4031051108,"1356998400000":-3383.977554232},"Current Account - Income - Investment income - Interests - Credit":{"694224000000":2014.0014714675,"725846400000":2027.2775502625,"757382400000":3002.312666,"788918400000":3833.8775029375,"820454400000":3973.129920925,"852076800000":4627.4401367958,"883612800000":5257.0804299068,"915148800000":5476.5362781333,"946684800000":6431.4841399625,"978307200000":4689.0736819,"1009843200000":2653.2796619441,"1041379200000":2615.0760793713,"1072915200000":2817.6832484494,"1104537600000":3238.3921593696,"1136073600000":3992.0192410357,"1167609600000":5071.8875776803,"1199145600000":4201.9037393871,"1230768000000":2140.3140600086,"1262304000000":1715.4233551163,"1293840000000":1878.3887297,"1325376000000":1577.0274336123,"1356998400000":1679.5572510774},"Current Account - Income - Investment income - Interests - Debit":{"694224000000":3494.4787706886,"725846400000":3608.7132591341,"757382400000":4774.2279814365,"788918400000":6374.5845782772,"820454400000":7353.1026375625,"852076800000":8826.4812929876,"883612800000":10346.6214808591,"915148800000":11329.0316956937,"946684800000":12352.0280282014,"978307200000":12161.7822847871,"1009843200000":10414.3374280747,"1041379200000":9999.0635093213,"1072915200000":9909.1752341243,"1104537600000":6816.4647128439,"1136073600000":5322.0942072569,"1167609600000":5855.8615285234,"1199145600000":5721.6122701467,"1230768000000":4526.5088816559,"1262304000000":4850.5619531139,"1293840000000":4933.5516597031,"1325376000000":5077.4305387231,"1356998400000":5063.5348053094},"Current Account - Income - Investment income - Profits and dividends":{"694224000000":-924.1772594016,"725846400000":-1501.1142312933,"757382400000":-1799.3376546582,"788918400000":-2093.8651789512,"820454400000":-2079.9905538226,"852076800000":-1919.9351380591,"883612800000":-2293.077219107,"915148800000":-1615.8012021599,"946684800000":-1609.2701389725,"978307200000":-258.232099422,"1009843200000":230.3393819369,"1041379200000":-633.0504924553,"1072915200000":-2287.4347212027,"1104537600000":-3894.6007541023,"1136073600000":-4939.3002864893,"1167609600000":-5241.1282638264,"1199145600000":-6093.9658052069,"1230768000000":-6627.4707239623,"1262304000000":-10730.2594225074,"1293840000000":-10734.4764316741,"1325376000000":-9193.0267546268,"1356998400000":-7578.3819551842},"Current Account - Income - Investment income - Profits and dividends - Credit":{"694224000000":335.0782192813,"725846400000":489.1018823375,"757382400000":459.764641,"788918400000":525.1029440875,"820454400000":460.726767675,"852076800000":857.2412421375,"883612800000":868.5795452125,"915148800000":597.5755707034,"946684800000":978.4226294741,"978307200000":634.7697690255,"1009843200000":350.3651438589,"1041379200000":451.292285908,"1072915200000":861.2727575841,"1104537600000":1022.695572617,"1136073600000":1637.530069811,"1167609600000":1487.3692413809,"1199145600000":1323.8659874411,"1230768000000":1291.5962636162,"1262304000000":940.4004805722,"1293840000000":1225.2210843483,"1325376000000":620.2475333583,"1356998400000":817.7142789153},"Current Account - Income - Investment income - Profits and dividends - Debit":{"694224000000":1259.2554786828,"725846400000":1990.2161136308,"757382400000":2259.1022956582,"788918400000":2618.9681230387,"820454400000":2540.7173214976,"852076800000":2777.1763801966,"883612800000":3161.6567643195,"915148800000":2213.3767728633,"946684800000":2587.6927684465,"978307200000":893.0018684475,"1009843200000":120.025761922,"1041379200000":1084.3427783633,"1072915200000":3148.7074787867,"1104537600000":4917.2963267193,"1136073600000":6576.8303563003,"1167609600000":6728.4975052073,"1199145600000":7417.8317926481,"1230768000000":7919.0669875785,"1262304000000":11670.6599030797,"1293840000000":11959.6975160225,"1325376000000":9813.2742879851,"1356998400000":8396.0962340995},"Current Account - Income - Other Income":{"694224000000":11.0,"725846400000":10.7,"757382400000":10.5,"788918400000":12.3,"820454400000":9.8,"852076800000":-3.6,"883612800000":-4.5,"915148800000":11.1,"946684800000":8.1436585899,"978307200000":4.2,"1009843200000":-24.9167570569,"1041379200000":-25.5020685572,"1072915200000":-39.2258185124,"1104537600000":-51.3604249553,"1136073600000":-59.5730231744,"1167609600000":-71.8840731085,"1199145600000":-57.9147709202,"1230768000000":-67.773329071,"1262304000000":-62.1674874722,"1293840000000":-54.4054560836,"1325376000000":-64.7809070834,"1356998400000":-48.9569078463},"Current Account - Current Transfers":{"694224000000":789.2,"725846400000":552.2,"757382400000":497.2,"788918400000":596.8,"820454400000":482.2,"852076800000":514.8575718422,"883612800000":464.068201997,"915148800000":453.3100648396,"946684800000":399.0359479504,"978307200000":424.4948174513,"1009843200000":540.0084772497,"1041379200000":504.1124759173,"1072915200000":561.0008299931,"1104537600000":483.8806229907,"1136073600000":459.4321818964,"1167609600000":353.4307110433,"1199145600000":169.6553899385,"1230768000000":2709.6831177935,"1262304000000":-405.6893477983,"1293840000000":-547.301686764,"1325376000000":-480.695954529,"1356998400000":-789.1327576839},"Capital and Financial Account":{"694224000000":9169.0772305226,"725846400000":14195.934222562,"757382400000":13781.418149039,"788918400000":7701.2956945017,"820454400000":12248.8335000228,"852076800000":17709.0904679406,"883612800000":18353.6981014051,"915148800000":13772.1331042733,"946684800000":8731.5564887885,"978307200000":-5441.5486050656,"1009843200000":-11339.2133225207,"1041379200000":-3105.2004919453,"1072915200000":1709.7188323803,"1104537600000":3689.9512572587,"1136073600000":-5221.3033783429,"1167609600000":5841.0975231075,"1199145600000":-7692.710359717,"1230768000000":-9184.5579326943,"1262304000000":7338.8444865578,"1293840000000":365.7271035774,"1325376000000":1041.2443597684,"1356998400000":-1316.1668773952},"Capital and Financial Account - Capital Account":{"694224000000":15.7,"725846400000":16.1,"757382400000":17.5,"788918400000":14.2,"820454400000":50.8,"852076800000":66.3,"883612800000":72.9,"915148800000":149.1,"946684800000":105.9,"978307200000":156.5,"1009843200000":406.11,"1041379200000":70.055,"1072915200000":196.3,"1104537600000":88.9,"1136073600000":97.1,"1167609600000":121.0976964454,"1199145600000":180.931,"1230768000000":74.028,"1262304000000":89.035,"1293840000000":62.3226,"1325376000000":47.845,"1356998400000":33.333503},"Capital and Financial Account - Financial Account":{"694224000000":9153.3772305226,"725846400000":14179.834222562,"757382400000":13763.918149039,"788918400000":7687.0956945017,"820454400000":12198.0335000228,"852076800000":17642.7904679406,"883612800000":18280.7981014051,"915148800000":13623.0331042733,"946684800000":8625.6564887885,"978307200000":-5598.0486050656,"1009843200000":-11745.3233225207,"1041379200000":-3175.2554919453,"1072915200000":1513.4188323803,"1104537600000":3601.0512572587,"1136073600000":-5318.4033783429,"1167609600000":5719.9998266621,"1199145600000":-7873.641359717,"1230768000000":-9258.5859326943,"1262304000000":7249.8094865578,"1293840000000":303.4045035774,"1325376000000":993.3993597684,"1356998400000":-1349.5003803952},"Capital and Financial Account - Financial Account - Banking Sector":{"694224000000":1304.7777,"725846400000":-1694.983,"757382400000":1999.305,"788918400000":4692.09,"820454400000":-197.045,"852076800000":-909.2690769231,"883612800000":3505.2041538462,"915148800000":1329.9571940718,"946684800000":67.414684892,"978307200000":11587.7227360192,"1009843200000":-2572.6565521268,"1041379200000":-3000.9658591709,"1072915200000":-3097.107481462,"1104537600000":-4355.9782466656,"1136073600000":-10508.0148210791,"1167609600000":1583.2903045107,"1199145600000":1732.5117343542,"1230768000000":-969.435,"1262304000000":-2678.7679260137,"1293840000000":6899.8932403548,"1325376000000":-1648.1947278223,"1356998400000":-1154.8691892003},"Capital and Financial Account - Financial Account - Banking Sector - Central Bank":{"694224000000":288.0,"725846400000":-3064.0,"757382400000":444.0,"788918400000":1922.0,"820454400000":1003.0,"852076800000":-586.0,"883612800000":-512.0,"915148800000":-1033.0,"946684800000":762.1,"978307200000":10742.7,"1009843200000":-1807.77699,"1041379200000":-867.8451257332,"1072915200000":-1989.988481462,"1104537600000":-2922.0262466656,"1136073600000":-10399.8828210791,"1167609600000":1493.1723045107,"1199145600000":1822.5957343542,"1230768000000":-1403.79,"1262304000000":-2910.0,"1293840000000":5000.0,"1325376000000":-2000.0,"1356998400000":-2000.0},"Capital and Financial Account - Financial Account - Banking Sector - Other financial entities":{"694224000000":1016.7777,"725846400000":1369.017,"757382400000":1555.305,"788918400000":2770.09,"820454400000":-1200.045,"852076800000":-323.2690769231,"883612800000":4017.2041538462,"915148800000":2362.9571940718,"946684800000":-694.685315108,"978307200000":845.0227360192,"1009843200000":-764.8795621268,"1041379200000":-2133.1207334377,"1072915200000":-1107.119,"1104537600000":-1433.952,"1136073600000":-108.132,"1167609600000":90.118,"1199145600000":-90.084,"1230768000000":434.355,"1262304000000":231.2320739863,"1293840000000":1899.8932403548,"1325376000000":351.8052721777,"1356998400000":845.1308107997},"Capital and Financial Account - Financial Account - Nonfinancial Public Sector":{"694224000000":796.0925,"725846400000":7013.6075,"757382400000":3994.0595,"788918400000":5854.8632741432,"820454400000":9024.0171514815,"852076800000":8162.6654958405,"883612800000":9361.252670445,"915148800000":10885.914659547,"946684800000":8252.8654630808,"978307200000":-3388.0690114367,"1009843200000":3682.8781373752,"1041379200000":4707.8581052861,"1072915200000":5109.5008008686,"1104537600000":3645.6649815621,"1136073600000":3616.4324776416,"1167609600000":2564.3614872748,"1199145600000":-400.4739985065,"1230768000000":-603.8273003504,"1262304000000":2907.7275438753,"1293840000000":-2360.8598770733,"1325376000000":-3338.2016750974,"1356998400000":574.2657224588},"Capital and Financial Account - Financial Account - Nonfinancial Public Sector - National Government (5) (6)":{"694224000000":1351.7925,"725846400000":6375.6075,"757382400000":4367.0595,"788918400000":6324.8632741432,"820454400000":8755.0171514815,"852076800000":6805.6654958405,"883612800000":9525.252670445,"915148800000":10108.214659547,"946684800000":7408.9654630808,"978307200000":-2925.6143913075,"1009843200000":3747.7466476583,"1041379200000":4805.9659520634,"1072915200000":5178.3966428505,"1104537600000":3562.4334298643,"1136073600000":3127.9092089705,"1167609600000":2318.449038929,"1199145600000":-181.4226769611,"1230768000000":-503.7438080962,"1262304000000":1184.7685246362,"1293840000000":-3463.8045147621,"1325376000000":-3859.2403769357,"1356998400000":605.8131701834},"Capital and Financial Account - Financial Account - Nonfinancial Public Sector - Local Governments":{"694224000000":43.3,"725846400000":879.0,"757382400000":190.0,"788918400000":394.0,"820454400000":733.0,"852076800000":1733.0,"883612800000":148.8,"915148800000":1364.1,"946684800000":1276.4,"978307200000":-58.6217821379,"1009843200000":237.1475117735,"1041379200000":166.5500756247,"1072915200000":191.2748485753,"1104537600000":334.1736009454,"1136073600000":808.8798435543,"1167609600000":447.8990042564,"1199145600000":-102.1684019249,"1230768000000":-65.4588574392,"1262304000000":1783.9162423641,"1293840000000":1163.4551579138,"1325376000000":557.3249297133,"1356998400000":-145.1406203056},"Capital and Financial Account - Financial Account - Nonfinancial Public Sector - Companies and other":{"694224000000":-599.0,"725846400000":-241.0,"757382400000":-563.0,"788918400000":-864.0,"820454400000":-464.0,"852076800000":-376.0,"883612800000":-312.8,"915148800000":-586.4,"946684800000":-432.5,"978307200000":-403.8328379912,"1009843200000":-302.0160220566,"1041379200000":-264.657922402,"1072915200000":-260.1706905571,"1104537600000":-250.9420492477,"1136073600000":-320.3565748831,"1167609600000":-201.9865559107,"1199145600000":-116.8829196205,"1230768000000":-34.624634815,"1262304000000":-60.957223125,"1293840000000":-60.510520225,"1325376000000":-36.286227875,"1356998400000":113.593172581},"Capital and Financial Account - Financial Account - Nonfinancial Private Sector":{"694224000000":7052.5070305226,"725846400000":8861.209722562,"757382400000":7770.553649039,"788918400000":-2859.8575796415,"820454400000":3371.0613485413,"852076800000":10389.3940490232,"883612800000":5414.3412771139,"915148800000":1407.1612506545,"946684800000":305.3763408157,"978307200000":-13797.7023296481,"1009843200000":-12855.544907769,"1041379200000":-4882.1477380605,"1072915200000":-498.9744870263,"1104537600000":4311.3645223623,"1136073600000":1573.1789650946,"1167609600000":1572.3480348766,"1199145600000":-9205.6790955647,"1230768000000":-7685.3236323439,"1262304000000":7020.8498686962,"1293840000000":-4235.6288597042,"1325376000000":5979.795762688,"1356998400000":-768.8969136537},"Net Errors and Omissions":{"694224000000":-337.2226718999,"725846400000":-1736.644282397,"757382400000":-2117.9651789444,"788918400000":-2699.1234402109,"820454400000":-1611.4208058375,"852076800000":-2320.5565081012,"883612800000":-451.100059154,"915148800000":-661.6884935575,"946684800000":-216.4385961609,"978307200000":-2861.4161537408,"1009843200000":-1878.089643753,"1041379200000":-1387.0927617357,"1072915200000":532.9583292442,"1104537600000":111.9173125238,"1136073600000":1163.2897425672,"1167609600000":57.5230684407,"1199145600000":1065.0982737799,"1230768000000":-338.0348932004,"1262304000000":-1715.5198502558,"1293840000000":-2760.6734043859,"1325376000000":-3169.4675076056,"1356998400000":-5721.8942160414},"International Reserves Variation":{"694224000000":3274.1,"725846400000":4250.4,"757382400000":682.0,"788918400000":-102.0,"820454400000":3882.0,"852076800000":3273.0,"883612800000":3438.0,"915148800000":1200.8,"946684800000":-439.4,"978307200000":-12083.4,"1009843200000":-4515.6494129455,"1041379200000":3580.777587845,"1072915200000":5319.128633818,"1104537600000":8856.8531912825,"1136073600000":3530.1557317197,"1167609600000":13098.1542682803,"1199145600000":8.89,"1230768000000":1346.21,"1262304000000":4157.34,"1293840000000":-6107.89,"1325376000000":-3304.72,"1356998400000":-11823.77},"International Reserves Variation - BCRA International Reserves":{"694224000000":3105.1,"725846400000":4480.4,"757382400000":561.0,"788918400000":-69.0,"820454400000":3782.0,"852076800000":3062.0,"883612800000":3442.0,"915148800000":1093.2,"946684800000":-424.0,"978307200000":-12004.6,"1009843200000":-4436.6494129455,"1041379200000":3642.777587845,"1072915200000":5527.128633818,"1104537600000":8430.8531912825,"1136073600000":3960.1557317197,"1167609600000":14139.1542682803,"1199145600000":209.89,"1230768000000":1581.57,"1262304000000":4222.34,"1293840000000":-5813.89,"1325376000000":-3085.72,"1356998400000":-12689.77},"International Reserves Variation - Exchange rate adjustment":{"694224000000":-169.0,"725846400000":230.0,"757382400000":-121.0,"788918400000":33.0,"820454400000":-100.0,"852076800000":-211.0,"883612800000":4.0,"915148800000":-107.6,"946684800000":15.4,"978307200000":78.8,"1009843200000":79.0,"1041379200000":62.0,"1072915200000":208.0,"1104537600000":-426.0,"1136073600000":430.0,"1167609600000":1041.0,"1199145600000":201.0,"1230768000000":235.36,"1262304000000":65.0,"1293840000000":294.0,"1325376000000":219.0,"1356998400000":-866.0},"Imports CIF":{"694224000000":14981.7,"725846400000":16872.3,"757382400000":21675.1,"788918400000":20199.7,"820454400000":23855.1,"852076800000":30450.2,"883612800000":31377.4,"915148800000":25508.2,"946684800000":25280.5,"978307200000":20319.6,"1009843200000":8989.5,"1041379200000":13850.8,"1072915200000":22445.3,"1104537600000":28686.94,"1136073600000":34153.7,"1167609600000":44707.5,"1199145600000":57462.5,"1230768000000":38786.267,"1262304000000":56792.578074,"1293840000000":74319.34850318,"1325376000000":68020.282609,"1356998400000":73655.536673}}